### PR TITLE
Migrating to Maven build from Gradle build.

### PR DESCRIPTION
**Pull request by Darshini-Parikh93**
- Removed Gradle configuration files.
- Added Maven configuration files.
- Migrating to WAR packaging.
- Updated Jenkins build file.
- Migrating to checkstyle external configuration from internal
configuration file.
- Removing port number for hosted EPID verification service.
- Fixing proxy identification issue in docker entry-point script.
- Updated README.

Signed-off-by: Darshini Parikh <darshini.k.parikh@intel.com>

### CODE REVIEW

File: demo/docker-compose.yml [hunk 1]
```diff
@@ -101,7 +101,7 @@ services:
             # To use Verification Service in the Cloud change RENDEZVOUS_VERIFICATIONSERVICEHOST variable to point to proper address and set PROXY for SDO service if needed
             # Do not set http_proxy and https_proxy if you use local Verification Service
             # Non-production environment cloud verification sevice:
-            RENDEZVOUS_VERIFICATIONSERVICEHOST: https://verify.epid-sbx.trustedservices.intel.com:443
+            RENDEZVOUS_VERIFICATIONSERVICEHOST: https://verify.epid-sbx.trustedservices.intel.com
```
Comment (trbehera) [inline demo/docker-compose.yml:104]: This seems to be an unrelated change. What is the reason for this change?

File: src/main/java/org/sdo/rendezvous/exceptions/handler/DefaultSdoExceptionHandler.java [hunk 2]
```diff
@@ -47,7 +47,7 @@ public void handle415(HttpMediaTypeNotSupportedException exception) {
    * application/json into HttpServletResponse.
    *
    * @param exception is a HttpMessageNotReadableException which is thrown and forwarded to the
-   *     method when HttpMessageConverter.read(java.lang.Class<? extends T>,
+   *     method when HttpMessageConverter.read(java.lang.Class &lt;\? extends T&gt;,
```
Comment (trbehera) [inline src/main/java/org/sdo/rendezvous/exceptions/handler/DefaultSdoExceptionHandler.java:50]: Is this change intentional?

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# Copyright 2019 Intel Corporation
+# SPDX-License-Identifier: Apache 2.0
+
+info.build.version=1.9
+
+rendezvous.hmacSecret=efe08a5df188dac44a4534c9f8ca0aed1b7ef6003622a07bf8a331134b3c1af427a6d7ee298f25d7f911ae495a8356e4636c19ada91643bd0d8b4f7b2bb97156
+rendezvous.tOTokenExpirationTime=15
+rendezvous.verificationServiceHost=https://verify.epid-sbx.trustedservices.intel.com:443
```
Comment (trbehera) [inline src/main/resources/application.properties:8]: In docker-compose file, the port information has been removed from the variable definition.

File: src/test/java/org/sdo/rendezvous/repositories/JedisRepositoryTest.java [hunk 4]
```diff
@@ -43,7 +43,8 @@
   private static String MOCKED_RANDOM_SEQUENCE = "aaaaaaaaaa";
   private static String MOCKED_LOG_RECORD =
       "{\"guid\":\"00000000-0000-0000-0000-000000000003\","
-          + "\"ct\":\"Tue May 09 10:33:39 CEST 2017\",\"gid\":\"00000000000000000000000000001234\"}";
+          + "\"ct\":\"Tue May 09 10:33:39 CEST 2017\","
+             + "\"gid\":\"00000000000000000000000000001234\"}";
```
Comment (trbehera) [inline src/test/java/org/sdo/rendezvous/repositories/JedisRepositoryTest.java:47]: Improper indent, both lines 46 and 47 should have same level of indent.

File: src/test/java/org/sdo/rendezvous/testng/custom/logs/CustomLogs.java [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  private long executionTime = 0;
+
+  @Override
+  public void onTestFailure(ITestResult tr) {
+
+    testClassName = tr.getTestClass().toString();
+    testMethodName = tr.getMethod().getMethodName().toString();
+    executionTime = tr.getEndMillis() - tr.getStartMillis();
+
+    System.out.println(testClassName + " [TestMethod name=" + testMethodName + "] [" + executionTime
```
Comment (trbehera) [inline src/test/java/org/sdo/rendezvous/testng/custom/logs/CustomLogs.java:22]: Use log4j functionality instead of System.out.println.

File: demo/docker-compose.yml (hunk 1 above)
Comment (Darshini-Parikh93) [reply demo/docker-compose.yml:104]: Based on the URL (i.e.) HTTP / HTTPS respective port (80 / 443) is picked up automatically. No need to specify it explicitly.

File: src/main/java/org/sdo/rendezvous/exceptions/handler/DefaultSdoExceptionHandler.java (hunk 2 above)
Comment (Darshini-Parikh93) [reply src/main/java/org/sdo/rendezvous/exceptions/handler/DefaultSdoExceptionHandler.java:50]: Yes.

File: pom.xml [hunk 6]
```diff
@@ -0,0 +1,340 @@
...
+        </dependencies>
+        <executions>
+          <execution>
+            <id>checkstyle-check</id>
+            <phase>compile</phase>
+            <goals>
+              <goal>check</goal>
+            </goals>
+            <configuration>
+              <configLocation>${project.basedir}/config/checkstyle/google_checks.xml</configLocation>
```
Comment (prateekcr) [inline pom.xml:293]: Maven can pick up google_checks.xml directly. Can we remove the path, and the local google_chekcs.xml file itself?

Comment (Darshini-Parikh93) [reply pom.xml]: Done

File: src/test/java/org/sdo/rendezvous/repositories/JedisRepositoryTest.java (hunk 4 above)
Comment (Darshini-Parikh93) [reply src/test/java/org/sdo/rendezvous/repositories/JedisRepositoryTest.java]: Corrected

File: src/test/java/org/sdo/rendezvous/testng/custom/logs/CustomLogs.java (hunk 5 above)
Comment (Darshini-Parikh93) [reply src/test/java/org/sdo/rendezvous/testng/custom/logs/CustomLogs.java]: Done

File: src/main/resources/application.properties (hunk 3 above)
Comment (Darshini-Parikh93) [reply src/main/resources/application.properties]: Removed port number
